oat-sa/extension-tao-itemqti
# hotfix/TR-5000/eliminate-label-break

related to https://oat-sa.atlassian.net/browse/TR-5000

### Description 

Backporting the fix to November release of tao-community which targets `29.14.4.5` version

https://github.com/oat-sa/tao-community/blob/5c0cc143ee4b35b7beb466cdd14683512c78a8fe/composer.json#L39

### How to test 

Apply the version to local install and check the Eliminate label doesnt break into 2 lines in `ja-JP`
